cpp/ql/test/query-tests/Likely Bugs/Protocols/boostorg/asio/boost_simulation.hpp

```diff
@@ -0,0 +1,112 @@
+#define SSL_OP_ALL										 0x80000BFFU
+/* Removed from OpenSSL 1.1.0. Was 0x01000000L */
```

Comment:
I just want to check, is this a comment you wrote to explain the difference between the versions, or was it copied from a boost header file?

Reply:
Perhaps it came from the OpenSSL project? https://github.com/openssl/openssl/blob/e7fb44e7c3f7a37ff83a6b69ba51a738e549bf5c/include/openssl/ssl.h#L462

Reply:
Ah, good spot.  In that case we really shouldn't include it as it's a copy-paste.  Will remove it shortly...

Reply:
Done.